services/movie.py

```diff
@@ -6,16 +7,17 @@
 def get_movies(
     genres_ids: list[int] = None,
     actors_ids: list[int] = None,
+    title: str = None,
```

Comment:
If argument can be `None`, you should show it in type hint. Fix everywhere, please
```suggestion
    title: str | None = None,
```